contracts/supplementary/rebalancing-manager/BTCETHRebalancingManager.sol

```diff
@@ -269,8 +270,8 @@ contract BTCETHRebalancingManager {
 
         // Require that the allocation has changed more than 2% in order for a rebalance to be called
         require(
-            btcDollarAmount.mul(100).div(currentSetDollarAmount) >= MAJORITY_ALLOCATION_LOWER_BOUND ||
-            btcDollarAmount.mul(100).div(currentSetDollarAmount) < MINORITY_ALLOCATION_UPPER_BOUND,
+            btcDollarAmount.mul(100).div(currentSetDollarAmount) >= upperAllocationBound ||
```

Comment:
Can we call this `minimumUpperThreshold` and `maximumLowerThreshold` to match rebalance strategy terminology?